src/main/java/org/wildfly/security/auth/realm/FileSystemSecurityRealm.java

```diff
@@ -347,20 +347,26 @@ public SupportLevel getEvidenceVerifySupport(final Class<? extends Evidence> evi
             List<Credential> credentials = loadCredentials();
             for (Credential credential : credentials) {
                 if (credential.canVerify(evidenceType, algorithmName)) {
+                    ElytronMessages.log.tracef("FileSystemSecurityRealm - evidence verification SUPPORTED: type = [%s]  algorithm = [%s]  credentials = [%d]", evidenceType, algorithmName, credentials.size());
```

Comment:
Yeah the general configuration at parent level is why I think we can deffer adding the additional child levels till later as the existing configuration at the parent level will still be correct.

We can try the different levels if we want and see what they end up like but it always feels weird to me when we have a single messages class, then loads of instances with each one only using a small subset of the messages.